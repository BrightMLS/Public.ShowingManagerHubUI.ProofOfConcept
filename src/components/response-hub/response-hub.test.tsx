import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import ResponseHub from './response-hub'
describe('<ResponseHub /> Component', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(
      <ResponseHub isResponse={true} responseObject={null} />,
    )
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})

describe('<ResponseHub /> Component', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(
      <ResponseHub isResponse={false} responseObject={null} />,
    )
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
