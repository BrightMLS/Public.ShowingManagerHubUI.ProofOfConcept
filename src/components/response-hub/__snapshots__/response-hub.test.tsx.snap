// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResponseHub /> Component should render without crashing 1`] = `
<Styled(MuiBox)
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Typography))
    className="makeStyles-typoStyle-3"
  >
    Response from HUB:
  </WithStyles(ForwardRef(Typography))>
  <div
    className="makeStyles-inputType-2 makeStyles-inputType-4"
    dangerouslySetInnerHTML={
      Object {
        "__html": "null",
      }
    }
  />
</Styled(MuiBox)>
`;

exports[`<ResponseHub /> Component should render without crashing 2`] = `
<Styled(MuiBox)
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Typography))
    className="makeStyles-typoStyle-3"
  >
    Response from HUB:
  </WithStyles(ForwardRef(Typography))>
  <div
    className="makeStyles-inputType-2 makeStyles-inputType-5"
    dangerouslySetInnerHTML={
      Object {
        "__html": "",
      }
    }
  />
</Styled(MuiBox)>
`;
